src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -179,12 +182,8 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
       }
     }
   };
-  const createComment = async (isAnonimity, anonimityData) => {
-    let sendPostNotif = false;
-    if (page !== 'DetailDomainScreen') {
-      sendPostNotif = true;
-    }
 
+  const handleAddedComment = (isAnonimity, anonimityData) => {
     const commentWillBeAddedData = {
       ...defaultData,
       data: {...defaultData.data, text: textComment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [185-208]

The `createComment` function has been refactored and a new function `handleAddedComment` has been introduced. This change improves the readability and maintainability of the code by separating concerns.

```diff
- const createComment = async (isAnonimity, anonimityData) => {
+ const handleAddedComment = (isAnonimity, anonimityData) => {
    const commentWillBeAddedData = {
      ...defaultData,
      data: {...defaultData.data, text: textComment}
    }
    setTemporaryText('');
    setNewCommentList([...newCommentList, commentWillBeAddedData]);
+ };
+ 
+ const createComment = async (isAnonimity, anonimityData) => {
+   let sendPostNotif = false;
+   if (page !== 'DetailDomainScreen') {
+     sendPostNotif = true;
+   }
+ 
+   handleAddedComment(isAnonimity, anonimityData);
```

228:
The `scrollToEnd` function is now called with `{animated: true}`. This change will make the scroll animation smoother.

```diff
- scrollViewRef.current.scrollToEnd();
+ scrollViewRef.current.scrollToEnd({animated: true});
```

291:
The `curHeight` variable has been added to the returned object. Ensure that this variable is used appropriately in the component where this hook is used.

```diff
+ curHeight,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->